src/pfe/portal/modules/User.js

```diff
@@ -469,6 +469,15 @@ module.exports = class User {
       projectID: projectID,
       status: 'success'
     };
+
+    // Delete files from PFE's /codewind-workspace/cw-temp/<project> location
+    const cwTempPath = path.join(project.workspace, "cw-temp", project.directory);
+    try {
+      await cwUtils.forceRemove(cwTempPath);
+    } catch (err) {
+      log.info(`Unable to delete temp files from ${cwTempPath}`)
+    }
+
```

Comment:
I think we ought to add this to deleteProjectFiles.  That does the rest of the file cleanup and removes the project from the project list so it can no longer be queried or have files uploaded to it. Deleting the temporary space before we do that sounds like we could end up with a hard to figure out bug later on.

Reply:
I thought about that, but it seems there was a chance exceptions could get thrown before deleteProjectFiles gets called. That would leave the temp files lingering wouldn't it ?

Reply:
Moved code to deleteProjectFiles to ensure filewatcher is done with the files before deleting temp